docs/vision-usage.rst

```diff
@@ -163,52 +103,72 @@ locations if available.
     >>> landmarks[0].bounding_poly.vertices[0].y_coordinate
     162
 
+
 Logo Detection
 ~~~~~~~~~~~~~~
 
-Google Vision can also attempt to detect company and brand logos in images.
+With :meth:`detect_logos()`, you can identify brand logos in an image. Their
+shape and location in the image can be found by iterating through the detected
+logo's `vertices`.
 
 .. code-block:: python
 
     >>> from google.cloud import vision
     >>> client = vision.Client()
-    >>> image = client.image('./image.jpg')
+    >>> with open('./image.jpg', 'rb') as image_file:
+    ...     image = client.image(content=image_file.read())
     >>> logos = image.detect_logos(limit=1)
-    >>> results.logos[0].description
+    >>> first_logo = logos[0]
+    >>> first_logo.description
     'Google'
-    >>> logos[0].score
+    >>> first_logo.score
     0.9795432
-    >>> logos[0].bounding_poly.vertices[0].x_coordinate
+    >>> first_logo.bounding_poly.vertices[0].x_coordinate
     78
-    >>> logos[0].bounding_poly.vertices[0].y_coordinate
+    >>> first_logo.bounding_poly.vertices[0].y_coordinate
     62
 
+
 Safe Search Detection
 ~~~~~~~~~~~~~~~~~~~~~
 
-Detecting safe search properties of an image.
+:meth:`detect_safe_search()` will try can categorize the entire contents of the
+image under four categories.
+
+- adult: Represents the adult contents likelihood for the image.
+- spoof: Spoof likelihood. The likelihood that an obvious modification was made
+         to the image's canonical version to make it appear funny or offensive.
+- medical: Likelihood this is a medical image.
+- violence: Violence likelihood.
 
 .. code-block:: python
 
     >>> from google.cloud import vision
     >>> client = vision.Client()
-    >>> image = client.image('./image.jpg')
+    >>> with open('./image.jpg', 'rb') as image_file:
+    ...     image = client.image(content=image_file.read())
     >>> safe_search = image.detect_safe_search()
     >>> safe_search[0].adult
     'VERY_UNLIKELY'
+    >>> safe_search[0].spoof
+    'POSSIBLE'
     >>> safe_search[0].medical
-    'UNLIKELY'
+    'VERY_LIKELY'
+    >>> safe_search[0].violence
+    'LIKELY'
+
 
 Text Detection
 ~~~~~~~~~~~~~~
 
-Detecting text with ORC from an image.
+:meth:`detect_text()` performs OCR to find text in the image.
```

Comment:
Why did you choose "the image" over "an image"? "an image" seems to be what you want?